docs/sources/datasources/elasticsearch/_index.md

```diff
@@ -0,0 +1,196 @@
+---
+aliases:
+  - /docs/grafana/latest/datasources/elasticsearch/
+  - /docs/grafana/latest/features/datasources/elasticsearch/
+  - /docs/grafana/latest/data-sources/elasticsearch/
+description: Guide for using Elasticsearch in Grafana
+keywords:
+  - grafana
+  - elasticsearch
+  - guide
+menuTitle: Elasticsearch
+title: Elasticsearch data source
+weight: 325
+---
+
+# Elasticsearch data source
+
+Grafana ships with built-in support for Elasticsearch.
+You can make many types of queries to visualize logs or metrics stored in Elasticsearch, and annotate graphs with log events stored in Elasticsearch.
+
+This topic explains configuring and querying specific to the Elasticsearch data source.
+For general documentation on querying data sources in Grafana, see [Query and transform data]({{< relref "../../panels-visualization/query-transform-data/" >}}).
+
+For instructions on how to add a data source to Grafana, refer to the [administration documentation]({{< relref "/administration/data-source-management/" >}}).
+Only users with the organization administrator role can add data sources.
+Administrators can also [configure the data source via YAML]({{< relref "#provision-the-data-source" >}}) with Grafana's provisioning system.
+
+> **Note:** You can use [Grafana Cloud](/products/cloud/features/#cloud-logs) to avoid the overhead of installing, maintaining, and scaling your observability stack.
+> The free forever plan includes Grafana, 10K Prometheus series, 50 GB logs, and more.
+> [Create a free account to get started](https://grafana.com/auth/sign-up/create-user?pg=docs-grafana-install&plcmt=in-text).
+
+Once you've added the Elasticsearch data source, you can [configure it]({{< relref "#configure-the-data-source" >}}) so that your Grafana instance's users can create queries in its [query editor]{{< relref "./query-editor/" >}} when they [build dashboards]({{< relref "../../dashboards/build-dashboards/" >}}) and use [Explore]({{< relref "../../explore/" >}}).
```

Comment:
Something wrong with the query editor link 
![image](https://user-images.githubusercontent.com/10999/197766425-b7330d39-d205-4454-bade-c75849dce581.png)

Reply:
same issue looks to be in all data source docs